README.md

```diff
@@ -1,34 +1,49 @@
-# ruff-action
+# Ruff GitHub Action
 
-> [!NOTE]
->
-> This Action is a fork of
-> [chartboost/ruff-action](https://github.com/ChartBoost/ruff-action), which is
-> no longer maintained. The Action is largely unchanged, but will be overhauled
-> in a future major release.
+[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
+[![image](https://img.shields.io/pypi/v/ruff.svg)](https://pypi.python.org/pypi/ruff)
+[![image](https://img.shields.io/pypi/l/ruff.svg)](https://github.com/astral-sh/ruff-action/blob/main/LICENSE)
+[![image](https://img.shields.io/pypi/pyversions/ruff.svg)](https://pypi.python.org/pypi/ruff)
+[![Actions status](https://github.com/astral-sh/ruff-action/workflows/CI/badge.svg)](https://github.com/astral-sh/ruff-action/actions)
 
-A [GitHub Action](https://github.com/features/actions) to run
-[Ruff](https://github.com/astral-sh/ruff).
+## Overview
 
-This action is commonly used as a pass/fail test to ensure your repository stays
-clean, abiding the [Rules](https://docs.astral.sh/ruff/rules/) specified in your
-configuration. Though it runs `ruff`, the action can do anything `ruff` can (ex,
-fix).
+The Ruff GitHub Action enforces code cleanliness by running Ruff against your repository. This ensures adherence to the [Rules](https://docs.astral.sh/ruff/rules/) defined in your configuration file. Whether you need Ruff to lint, fix, or perform any of its robust features, this action has you covered.
 
 ## Compatibility
 
-This action is known to support all GitHub-hosted runner OSes. It likely can run
-on self-hosted runners, but might need specific dependencies. Only published
-versions of Ruff are supported (i.e., whatever is available on
-[PyPI](https://pypi.org/project/ruff/)).
+This action supports all GitHub-hosted runner operating systems. While it likely runs on self-hosted runners, be aware that additional dependencies may be required. Note that only published versions of Ruff from PyPI are supported.
 
-## Basic Usage
+## Usage
 
-Create a file (ex: `.github/workflows/ruff.yml`) inside your repository with:
+### Pre-requisites
+
+Create a workflow `.yml` file in your repository's `.github/workflows` directory. An [example workflow](#example-workflows) is available below. For more information, see the GitHub Help Documentation for [Creating a workflow file](https://help.github.com/en/articles/configuring-a-workflow#creating-a-workflow-file).
+
+### Inputs
+
+> [!WARNING]
+> Note that `isolated` and `config_path`, as well as `version` and `use_pyproject`, are mutually exclusive and cannot be configured simultaneously.
+
+- `args`: Arguments passed to Ruff. Use `ruff --help` to see available options. Default: `check`.
+- `src`: Source to run Ruff. Default: `'.'`.
+- `version`: The version of Ruff to use, e.g. "0.5.0". Default: `""`.
```

Comment:
```suggestion
- `version`: The version of Ruff to use, e.g. `"0.5.0"`. Default: `""`.
```
This may help reinforce writing the version with quotes. As writing it without is a footgun (ie: `0.10` is actually `0.1`)